modyn/storage/internal/grpc/storage_grpc_servicer.py

```diff
@@ -243,6 +245,25 @@ def GetDataPerWorker(
                 if len(batch) > 0:
                     yield GetDataPerWorkerResponse(keys=[value[0] for value in batch])
 
+    # pylint: disable-next=unused-argument,invalid-name
+    def GetDatasetSize(self, request: GetDatasetSizeRequest, context: grpc.ServicerContext) -> GetDatasetSizeResponse:
```

Comment:
Could you please also add an integration test for this? That would be great since we are rewriting the storage currently and we need to keep track end2end if the C++ implementation has the same behavior as the Python implementation :) Thank you!

Reply:
Sure :) done in next commit.

Reply:
thank you!